boards/com.matternet.bmu_4.3/board.ld

```diff
@@ -0,0 +1,114 @@
+MEMORY
+{
+    bl (rx) :             ORIGIN = 0x08000000,            LENGTH = 12K
+    app (rx) :            ORIGIN = 0x08000000+12K,        LENGTH = 128K-12K-2K
+    params (rx) :         ORIGIN = 0x08000000+126K,        LENGTH = 2K
+    ram (rwx) :           ORIGIN = 0x20000000,            LENGTH = 32K-256
+    app_bl_shared (rwx) : ORIGIN = 0x20000000+(32K-256),  LENGTH = 256
+}
+
+
+REGION_ALIAS("PROGRAM_REGION", bl)
+
+/* Enforce emmition of the vector table. */
+EXTERN (vector_table)
+
+/* Define the entry point of the output file. */
+ENTRY(reset_handler)
+
+/* Define sections. */
+SECTIONS
+{
+    .text : {
+        *(.vectors)	/* Vector table */
+        KEEP(*(.app_descriptor))
+        *(.text*)	/* Program code */
+        . = ALIGN(4);
+        *(.rodata*)	/* Read-only data */
+        . = ALIGN(4);
+        _otp = ALIGN(4);
+    } >PROGRAM_REGION
+
+    .bl (NOLOAD) : {
+    } >bl
+
+    .app (NOLOAD) : {
+    } >app
+
+    .params(NOLOAD) : {
+    } >params
+
+    .app_bl_shared (NOLOAD) : {
+    } >app_bl_shared
+
+    /* C++ Static constructors/destructors, also used for __attribute__
+        * ((constructor)) and the likes */
+    .preinit_array : {
+        . = ALIGN(4);
+        __preinit_array_start = .;
+        KEEP (*(.preinit_array))
+        __preinit_array_end = .;
+    } >PROGRAM_REGION
+    .init_array : {
+        . = ALIGN(4);
+        __init_array_start = .;
+        KEEP (*(SORT(.init_array.*)))
+        KEEP (*(.init_array))
+        __init_array_end = .;
+    } >PROGRAM_REGION
+    .fini_array : {
+        . = ALIGN(4);
+        __fini_array_start = .;
+        KEEP (*(.fini_array))
+        KEEP (*(SORT(.fini_array.*)))
+        __fini_array_end = .;
+    } >PROGRAM_REGION
+
+    /*
+        * Another section used by C++ stuff, appears when using newlib with
+        * 64bit (long long) printf support
+        */
+    .ARM.extab : {
+        *(.ARM.extab*)
+    } >PROGRAM_REGION
+    .ARM.exidx : {
+        __exidx_start = .;
+        *(.ARM.exidx*)
+        __exidx_end = .;
+    } >PROGRAM_REGION
+
+    . = ALIGN(4);
+    _etext = .;
+
+    .data : {
+        _data = .;
+        *(.data*)	/* Read-write initialized data */
+        . = ALIGN(4);
+        _edata = .;
+    } >ram AT >PROGRAM_REGION
+    _data_loadaddr = LOADADDR(.data);
+
+    .bss : {
+        *(.bss*)	/* Read-write zero initialized data */
+        *(COMMON)
+        . = ALIGN(4);
+        _ebss = .;
+    } >ram
+
+    /*
+        * The .eh_frame section appears to be used for C++ exception handling.
+        * You may need to fix this if you're using C++.
+        */
```

Comment:
Same comment here as above regarding the block comment start / end placement :)

Reply:
Done!